### PR TITLE
Introduce option pic

### PR DESCRIPTION
**Pull request by levitte**
For display and to see how Travis likes the result (I should probably add a config where we turn off pic)

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -5,13 +5,17 @@
...
-     This only applies to the engines in engines/, those in
-     crypto/engine/ will always be built into libcrypto (i.e. "static").
+     without having to build shared libraries and vice versa.  This
+     only applies to the engines in engines/, those in crypto/engine/
+     will always be built into libcrypto (i.e. "static").
+
+     Building dynamic engines is enabled by default; to disable, use
+     the configuration option "disable-dynamic-engine".
+
+     The only requirements for building dynamic engines is the
```
Comment (jrchamp) [inline CHANGES]: "is" => "are"

File: CHANGES [hunk 2]
```diff
@@ -5,13 +5,17 @@
...
+     presence of the DSO module and building with position independent
+     code, so they will also automatically be disabled if configuring
+     with "disable-dso" or "disable pic".
```
Comment (jrchamp) [inline CHANGES]: Should this be disable-pic?

**Comment by levitte** [comment]
Has now been merged.  Closing

**Comment by levitte** [comment]
Sorry, @jrchamp, didn't see those before now.  Corrections coming up

File: Configure [hunk 3]
```diff
@@ -46,6 +46,8 @@ my $usage="Usage: Configure [no-<cipher> ...] [enable-<cipher> ...] [-Dxxx] [-lx
 #               multithreaded applications (default is "threads" if we
 #               know how to do it)
 # [no-]shared	[don't] try to create shared libraries when supported.
+# [no-]pic      [don't] try to build position independent code when supported.
+#               If disabled, it also disables shared and dynamic-engines.
```
Comment (jrchamp) [inline Configure]: It looks like dynamic-engine is singular in the other changes.

**Comment by levitte** [comment]
Was that all, btw, or do you have more?

File: Configure (hunk 3 above)
Comment (levitte) [inline Configure]: True

**Comment by jrchamp** [comment]
I'm done now, sorry!

**Comment by levitte** [comment]
No trouble at all, thanks for pointing those out.  I'm sorry for being trigger happy at times ;-)

**Comment by jrchamp** [comment]
No worries, I'm glad it helped. :smile:

**Pull request closed without merging**